test/util/TestData.scala

```diff
@@ -101,7 +101,7 @@ object TestData {
     NodeSeq.Empty,
     FakeRequest().withJsonBody(Json.obj("fake" -> "request")),
     BatchFileUploadRequest(DeclarationId("decId123"),FileGroupSize(2),
```

Comment:
how about `Seq(BatchFileUploadFile(FileSequenceNo(1), maybeDocumentType = None)` - where we have `None` as a parameter maybe we should used named parameters (bot not for the `Some` values as they describe themselves)